tox.ini

```
@@ -29,9 +29,10 @@ deps =
     mongo: mongomock
     # mongomock imports pkg_resources, provided by setuptools.
     mongo: setuptools>=66.1.1
-    django{42,51,main}: Pillow
+    django{42,51,52,main}: Pillow
     django42: Django>=4.2,<5.0
     django51: Django>=5.1,<5.2
+    django52: Django>=5.2,<6
     djangomain: Django>5.1,<6.0
```

This one should probably be bumped too? Otherwise it might be testing the same as above... We might have to [revert to downloading from source](https://github.com/FactoryBoy/factory_boy/commit/6f5f6a05dec30a3d909522f2667a90240665abfb) as [there is no 6.0](https://pypi.org/project/Django/#history) on PyPI yet:

```suggestion
    djangomain: https://github.com/django/django/archive/main.tar.gz
```

---

While I completely agree with your suggestion, I’m not sure this change should be included in this PR. It’s unrelated to the scope of the current work and would be better handled in a separate PR. I’m a strong proponent of keeping PR titles and bodies aligned with their actual changes.

PS. This sounds like a great idea for a contribution from you @browniebroke -wink-wink-

---

Well, you're argument makes _some_ sense... However, I'm going to make another one which may also makes sense (I think):

This PR adds support to a new version of Django, and doing so means that "django main" should keep pointing to the next version, which means updating this 😄

Anyway, I'll leave it up to a maintainer to decide. There is a gap in my understanding as to why it was changed away from installing from source in the first place, maybe they can shed some light on that.